src/python/test/file_load/job_identity_token/dxapp.json

```diff
@@ -0,0 +1,51 @@
+{ "name": "basic",
+  "title": "basic",
+  "summary" : "basic file upload/download",
+  "runSpec": {
+    "file": "run.sh",
+    "interpreter": "bash",
+    "distribution": "Ubuntu",
+    "release": "20.04",
+    "version": "0"
+  },
+  "inputSpec": [
+    {"name": "audience",  "class": "string"},
+    {"name": "subject_claims",  "class": "array:string", "optional" : true}
```

Comment:
I would consider removing optional as if not provided, you'd get `subject_claims = ['']` which is not valid request